azure-devops/azext_devops/dev/common/identities.py

```diff
@@ -116,5 +129,13 @@ def get_account_from_identity(identity):
     return identity.provider_display_name
 
 
+def get_identity_descriptor_from_subject_descriptor(subject_descriptor, organization):
+    identity_client = get_identity_client(organization)
+    identities = identity_client.read_identities(subject_descriptors=subject_descriptor)
```

Comment:
It is recommended to perform basic validation on request parameter subject_descriptor before making the identities call.

Reply:
Validation checks are done before making this call at permissions side, since Email IDs can also have '.' in them.

Reply:
Makes sense in our case. My comment was generic from a perspective of having test cases for a public method.